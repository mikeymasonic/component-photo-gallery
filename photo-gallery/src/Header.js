import React, { Component } from "react";

export default class Header extends Component {
  render() {
    return (
      <div>
            {/* <img src={this.props.logoText} /> */}
        <header>
          <img className="logo" src="logo.png" alt="Horn" />
          <h1>Various Horned Animals</h1>
        </header>
        
      </div>
    );
  }
}
