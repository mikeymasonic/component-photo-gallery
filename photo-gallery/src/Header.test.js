import React from 'react';
// import { render } from '@testing-library/react';
// import App from './App';
import Header from './Header'
import renderer from "react-test-renderer";
test('renders App.js correctly', () => {
  const tree = renderer
  .create(<Header />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});