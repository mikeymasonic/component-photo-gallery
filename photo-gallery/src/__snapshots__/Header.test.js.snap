// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App.js correctly 1`] = `
<div>
  <header>
    <img
      alt="Horn"
      className="logo"
      src="logo.png"
    />
    <h1>
      Various Horned Animals
    </h1>
  </header>
</div>
`;
